src/orientation.rs

```diff
@@ -0,0 +1,40 @@
+/// Describes the transformations to be applied to the image.
+/// Compatible with [Exif orientation](https://web.archive.org/web/20200412005226/https://www.impulseadventure.com/photo/exif-orientation.html).
+///
+/// Orientation is specified in the Exif metadata, and is often written by cameras.
+#[derive(Copy, Clone, PartialEq, Eq, Hash, Debug)]
+pub enum Orientation {
+    /// Do not perform any transformations.
+    NoTransforms,
+    /// Rotate by 90 degrees clockwise.
+    Rotate90,
+    /// Rotate by 180 degrees. Can be performed in-place.
+    Rotate180,
+    /// Rotate by 270 degrees clockwise. Equivalent to rotating by 90 degrees counter-clockwise.
+    Rotate270,
+    /// Flip horizontally. Can be performed in-place.
+    FlipHorizontal,
+    /// Flip vertically. Can be performed in-place.
+    FlipVertical,
+    /// Rotate by 90 degrees clockwise and flip horizontally.
+    Rotate90FlipH,
```

Comment:
It is a bit long, but perhaps `Rotate90FlipHorizontal` for consistency?

Reply:
I tried that, and my eyes just glaze over when I try to read it. I genuinely believe this option is more readable. 

And if its meaning is ever unclear, there is always the doc comment.